source-release/pom.xml

```diff
@@ -101,6 +101,14 @@ under the License.
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <configuration>
+            <excludes>
+              <exclude>**/maven-projects/**</exclude>
```

Comment:
Is it always `maven-projects`? I have also seen `projects` as ITs in several modules.

Reply:
I think my question can be scratched. The issue applies to this project only.